### PR TITLE
Blog post on the Cake for Rider release 0.3.0

### PR DESCRIPTION
**Pull request by nils-a**

### CODE REVIEW

File: input/blog/2021-04-01-cake-rider-0.3.0-released.md
Comment (augustoproiete) [inline input/blog/2021-04-01-cake-rider-0.3.0-released.md:9]: Small suggestion:

```suggestion
Highlights of this release include:
```

**Review by augustoproiete — APPROVED** [review]
LGTM

File: input/blog/2021-04-01-cake-rider-0.3.0-released.md
Comment (nils-a) [reply input/blog/2021-04-01-cake-rider-0.3.0-released.md]: 👍

**Comment by nils-a** [comment]
[Cake for Rider 0.3.0](https://github.com/cake-build/cake-rider/releases/tag/0.3.0) has been submitted and is now in moderation. I will merge this PR once the package habe been approved by JetBrains.

**Pull request merged**
